app/routes/synonyms/synonyms.server.ts

```diff
@@ -53,7 +53,6 @@ export async function searchSynonymsByEventId({
   items: SynonymGroup[]
   totalItems: number
   totalPages: number
-  page: number
```

Comment:
Please restore this argument. Removing the `page` argument will break pagination by causing page changes to have no effect.

Reply:
Can you explain this a little more? What that is doing is saying that it returns a promise with a page field. If I remove the page from the return, shouldn't I also remove it from where the shape of what I'm returning is defined? How does removing it from the promise affect pagination?

Reply:
Sorry, I misread.